### ISSUE TITLE
Pre-work Checklist: Developer: Karen G. Cecena

### ISSUE THREAD

**Issue by karengcecena** [hackforla/website#4072]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by karengcecena** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings? About 5 hours.

**Comment by jefflueck** [comment]
It only took me about an hour to complete the issue I assigned myself to. It took me longer to figure out the proper way to push up the change and submit a pull request.

Respectfully,

Jeff Lueck

**Comment by ExperimentsInHonesty** [comment]
@karengcecena I am reopening this issue until you have been assigned to a development issue long enough to make an in progress update.  It can will stay open until you get an issue that takes longer than a week to complete (likely a medium or large size issue).  That way, we know you can make detailed progress updates.  In the meantime, if you get reminders to update this issue, just let us know what size and number issue you are assigned to at the time.  That way, we have a record (in one place) of your progress.

I see you finished your good first issue and second issue but have not yet picked up a small issue.  Here are the issues that are available that match the role labels you choose on your prework.  Tthere is currently one of each, but I did send this message out to several people, so if there are none when you check back, let the merge team know, and reply on this issue.
[Front End, Small](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-7198257) | [Back End, Small](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257)

**Comment by karengcecena** [comment]
Progress: No Current Project
Blockers: Having a hard time finding an issue I can complete that isn't already taken
Availability: Friday afternoon, Saturday morning
ETA: N/A

**Comment by ExperimentsInHonesty** [comment]
@karengcecena 
- I am going to unassign you from https://github.com/hackforla/website/issues/4211 issue because you are not available until Wed and this issue is time sensitive. So it's better to leave it for someone else to pick up that is available now.  Additionally, please don't pick up issues if you are not available for 5 days.  Pick up an issue when you are available (if there is one to pick up at the time).

The merge team is working hard to make more issues available.

When you are ready again, please check these links

[Small backend](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257)

[Small front end](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-7198257)

**Comment by karengcecena** [comment]
Progress: No Current Project
Blockers: I had a hard time finding an issue I could complete earlier in the week and my family got covid later in the week so I've had to prioritize taking care of them. 
Availability: N/A
ETA: N/A

**Comment by karengcecena** [comment]
Progress: I have completed a small issue and have been reviewing multiple pull requests. 
Blockers: I am having a hard time finding a medium issue for either front or back end. 
Availability: Monday - Friday, 7am - 3pm
ETA: N/A

**Comment by karengcecena** [comment]
1. Progress: I have been working on medium issue #4469 and  have been reviewing pull requests.
2. Blockers: None, just waiting on a review of my current step in the medium issue
3. Availability: From 7:30am - 2:30pm on April 24-28
4. ETA: April 28 EOD

**Comment by karengcecena** [comment]
1. Progress: I have been waiting on feedback on my medium issue https://github.com/hackforla/website/issues/4469 and have been reviewing pull requests in the mean time.
2. Blockers: None, just waiting on a review of my current step in the medium issue
3. Availability: From 7:30am - 2:30pm Mon-Fri
4. ETA: May 19th?

**Comment by karengcecena** [comment]
1. Progress: I have been waiting on feedback on my medium issue https://github.com/hackforla/website/issues/4469. I messaged Justin, who referred me to Roslyn, Sarah, and Will. I messaged them as well and we are now trying to set up a time to discuss the issue. In the meantime, I created an [ER for the current GHA documentation](https://github.com/hackforla/website/issues/4728). 
2. Blockers: n/a
3. Availability: From 7:30am - 2:30pm Mon-Fri EXCEPT I will be unavailable Thursday, May 25 - Tuesday, May 30th. 
4. ETA: June 9th, 2023?

**Comment by karengcecena** [comment]
Progress: I received feedback on my medium issue https://github.com/hackforla/website/issues/4469. I plan to implement the suggestions in the next week.
Blockers: N/A
Availability: Monday, June 5, 2023 from 9am-12pm
ETA:  Monday, June 5 , 2023 at 12pm 
Pictures (optional): N/A

**Comment by karengcecena** [comment]
Progress: I implemented the feedback I received on my medium issue https://github.com/hackforla/website/issues/4469. I am waiting to hear back again before I can start creating new issues for each GHA. 
Blockers: N/A
Availability: Monday-Tuesday, June 19-20, 2023, 9am-5pm.
ETA: 2 weeks from now (~ June 30th, 2023)

**Comment by roslynwythe** [comment]
Hi @karengcecena we appreciate your previous updates.  For the time being we are asking developers to leave an update at least every two weeks (using the outline in the above comment) to avoid the `2 weeks inactive` label, which flags the issue for review by a dev lead.  We are working on improvements to the prework checklist so that comments will automatically posted when developers open/close issues and PRs, and so if developers are working on other issues, it will not be necessary to leave updates in this issue.    

Thank you again for your contributions to Hack for LA.

**Comment by karengcecena** [comment]
Hi @roslynwythe, I'm sorry I missed the update notification. I hadn't realized the other issue I am working on had been commented on. 

Progress: Planning to go to the meeting this Sunday to discuss the issue https://github.com/hackforla/website/issues/4469
Blockers: Time Management & Communication
Availability: 4 hours
ETA: July 16, 2023

**Comment by karengcecena** [comment]
Progress: I was having a hard time finding a dev time I could make so I unassigned myself from issue: https://github.com/hackforla/website/issues/4469
Blockers: Time
Availability: N/A
ETA: N/A

**Comment by t-will-gillis** [comment]
Inactive member; closed due to inactivity

**Comment by HackforLABot** [comment]
## Activity Log: karengcecena
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- karengcecena opened: https://github.com/hackforla/website/issues/4072 at 02/28/2023, 08:28 PM PST
- karengcecena assigned: https://github.com/hackforla/website/issues/4072 at 03/01/2023, 01:43 PM PST
- karengcecena assigned: https://github.com/hackforla/website/issues/4049#issuecomment-1454852886 at 03/05/2023, 07:50 AM PST
- karengcecena commented on: https://github.com/hackforla/website/issues/4049#issuecomment-1455134780 at 03/05/2023, 08:13 AM PST
- karengcecena commented on: https://github.com/hackforla/website/issues/4072#issuecomment-1455139728 at 03/05/2023, 08:32 AM PST
- karengcecena opened: https://github.com/hackforla/website/pull/4114 at 03/05/2023, 08:47 AM PST
- karengcecena PR was merged: https://github.com/hackforla/website/pull/4114#event-8680283053 at 03/06/2023, 04:33 PM PST
- karengcecena issue closed by PR 4114: https://github.com/hackforla/website/issues/4049#event-8680283206 at 03/06/2023, 04:33 PM PST
- karengcecena closed as completed: https://github.com/hackforla/website/issues/4072#event-8700109350 at 03/08/2023, 12:03 PM PST
- karengcecena commented on: https://github.com/hackforla/website/pull/4144#issuecomment-1465027105 at 03/11/2023, 01:39 PM PST
- karengcecena commented on: https://github.com/hackforla/website/pull/4144#issuecomment-1465037212 at 03/11/2023, 02:21 PM PST
- karengcecena assigned: https://github.com/hackforla/website/issues/4000 at 03/14/2023, 10:01 AM PDT
- karengcecena opened: https://github.com/hackforla/website/pull/4179 at 03/14/2023, 10:19 AM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4000#issuecomment-1468523050 at 03/14/2023, 10:25 AM PDT
- karengcecena commented on: https://github.com/hackforla/website/pull/4176#issuecomment-1471228633 at 03/15/2023, 08:20 PM PDT
- karengcecena submitted review: https://github.com/hackforla/website/pull/4176#pullrequestreview-1342812172 at 03/15/2023, 08:25 PM PDT
- karengcecena PR was merged: https://github.com/hackforla/website/pull/4179#event-8762721366 at 03/15/2023, 09:22 PM PDT
- karengcecena issue closed by PR 4179: https://github.com/hackforla/website/issues/4000#event-8762721485 at 03/15/2023, 09:22 PM PDT
- karengcecena commented on: https://github.com/hackforla/website/pull/4214#issuecomment-1475460762 at 03/19/2023, 05:42 PM PDT
- karengcecena submitted review: https://github.com/hackforla/website/pull/4214#pullrequestreview-1347640319 at 03/19/2023, 05:47 PM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4072#issuecomment-1483073773 at 03/24/2023, 09:20 AM PDT
- karengcecena assigned: https://github.com/hackforla/website/issues/4211 at 03/25/2023, 08:01 PM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4211#issuecomment-1483980811 at 03/25/2023, 08:04 PM PDT
- karengcecena unassigned: https://github.com/hackforla/website/issues/4211 at 03/26/2023, 08:25 PM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4072#issuecomment-1493055270 at 04/01/2023, 10:42 AM PDT
- karengcecena assigned: https://github.com/hackforla/website/issues/4258 at 04/11/2023, 07:26 AM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4258#issuecomment-1503471639 at 04/11/2023, 07:27 AM PDT
- karengcecena opened: https://github.com/hackforla/website/pull/4467 at 04/11/2023, 10:50 AM PDT
- karengcecena commented on: https://github.com/hackforla/website/pull/4474#issuecomment-1505376958 at 04/12/2023, 07:27 AM PDT
- karengcecena submitted review: https://github.com/hackforla/website/pull/4474#pullrequestreview-1381503472 at 04/12/2023, 07:51 AM PDT
- karengcecena commented on: https://github.com/hackforla/website/pull/4472#issuecomment-1505440317 at 04/12/2023, 08:05 AM PDT
- karengcecena submitted review: https://github.com/hackforla/website/pull/4472#pullrequestreview-1381551389 at 04/12/2023, 08:14 AM PDT
- karengcecena PR was merged: https://github.com/hackforla/website/pull/4467#event-8992048464 at 04/12/2023, 08:12 PM PDT
- karengcecena issue closed by PR 4467: https://github.com/hackforla/website/issues/4258#event-8992048582 at 04/12/2023, 08:12 PM PDT
- karengcecena commented on: https://github.com/hackforla/website/pull/4497#issuecomment-1507277981 at 04/13/2023, 09:45 AM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4072#issuecomment-1507282047 at 04/13/2023, 09:49 AM PDT
- karengcecena submitted review: https://github.com/hackforla/website/pull/4497#pullrequestreview-1383838870 at 04/13/2023, 09:50 AM PDT
- karengcecena commented on: https://github.com/hackforla/website/pull/4500#issuecomment-1507285882 at 04/13/2023, 09:52 AM PDT
- karengcecena commented on: https://github.com/hackforla/website/pull/4500#issuecomment-1507300344 at 04/13/2023, 10:00 AM PDT
- karengcecena submitted review: https://github.com/hackforla/website/pull/4500#pullrequestreview-1384373032 at 04/13/2023, 04:20 PM PDT
- karengcecena commented on: https://github.com/hackforla/website/pull/4500#issuecomment-1507723008 at 04/13/2023, 04:21 PM PDT
- karengcecena assigned: https://github.com/hackforla/website/issues/4469 at 04/17/2023, 02:28 PM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4469#issuecomment-1512110486 at 04/17/2023, 02:30 PM PDT
- karengcecena submitted review: https://github.com/hackforla/website/pull/4497#pullrequestreview-1391186885 at 04/18/2023, 07:45 PM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4469#issuecomment-1516986206 at 04/20/2023, 02:49 PM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4469#issuecomment-1516995724 at 04/20/2023, 03:00 PM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4072#issuecomment-1518010051 at 04/21/2023, 08:35 AM PDT
- karengcecena submitted review: https://github.com/hackforla/website/pull/4474#pullrequestreview-1398445423 at 04/24/2023, 09:52 AM PDT
- karengcecena submitted review: https://github.com/hackforla/website/pull/4474#pullrequestreview-1413994864 at 05/04/2023, 03:19 PM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4072#issuecomment-1546231004 at 05/12/2023, 01:08 PM PDT
- karengcecena opened: https://github.com/hackforla/website/issues/4728 at 05/25/2023, 07:42 AM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4072#issuecomment-1563151767 at 05/25/2023, 09:00 AM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4469#issuecomment-1574312603 at 06/02/2023, 02:03 PM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4072#issuecomment-1574315814 at 06/02/2023, 02:06 PM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4469#issuecomment-1582815815 at 06/08/2023, 08:25 AM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4469#issuecomment-1594996332 at 06/16/2023, 10:09 AM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4072#issuecomment-1595001479 at 06/16/2023, 10:14 AM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4469#issuecomment-1622955849 at 07/05/2023, 09:25 PM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4072#issuecomment-1622957858 at 07/05/2023, 09:27 PM PDT
- karengcecena commented on: https://github.com/hackforla/website/issues/4072#issuecomment-1646308090 at 07/21/2023, 03:35 PM PDT
